gridded_etl_tools/utils/extractor.py

```diff
@@ -187,6 +181,17 @@ class FTPExtractor(Extractor):
     # Used to limit retrieval to a single thread at a time
     semaphore: multiprocessing.synchronize.Semaphore = multiprocessing.Semaphore()
 
+    def __init__(self, host: str):
+        """
+        When called like a function, set the host parameter. See `FTPExtractor.__enter__`.
```

Comment:
Description needs to be updated and I don't think __enter__ needs to be referenced anymore